CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+0.5.6
+ - Added check file for changes function, it is calculating a MD5 hash for each file and saves it in input/DO_NOT_CHANGE_GRAPHQL.checksum file
+ - it recognizes if a user changed a file since the last add-type/update-type/remove-type and stops with a warning
+ - user can force update to overwrite even changed code files by using option force-update
+
+0.5.5
+ - Removed the archive folders from the skeleton
+ - Removed copy/move functions for former <type> files 
+ - Added checkFileForPattern, to figure out, if an index file already contains the <type> pattern
+ - Refactored the code, to get it more maintainable
+
+0.5.4
+
+ - Added archive folders to skel/schema, skel/resolver, skel/model to keep old versions of files for security reasons
+ - Added input folder to host the input <type>.graphql files, having an opinionated way to store them
+ - Added CMD option: update-type to move existing versions into there respective archive folders, regenerate new ones
+ - Added CMD option: remove-type to move existing versions into there respective archive folders
+ - Copied the two examples User.graphql, Tweet.graphql into the input folder as examples
+ - Added one npm dependency 'escape-string-regexp' for escaping code lines to be used in regular expressions
+
```

Comment:
Can coalesce these into a simple changelog that just describes the user facing changes. (Plus you can use git to squash your changes if you want to clean things up, lmk if I can help with that).